internal/signer/signer.go

```diff
@@ -26,11 +27,11 @@ var (
 )
 
 type serviceAccountCredentials struct {
-	Mrn         string `json:"mrn,omitempty"`
-	ParentMrn   string `json:"parent_mrn,omitempty"`
-	PrivateKey  string `json:"private_key,omitempty"`
-	Certificate string `json:"certificate,omitempty"`
-	ApiEndpoint string `json:"api_endpoint,omitempty"`
+	Mrn         string `json:"mrn,omitempty" yaml:"mrn,omitempty"`
```

Comment:
With the switch to https://github.com/kubernetes-sigs/yaml this change is not required.

Reply:
Will switch! Thanks, @chris-rock 👍🏽